README.md

```diff
@@ -3,160 +3,43 @@
 [SemVer]-compatible hooks for [`setuptools_scm`].
 
 This package was created because `setuptools_scm` produces [PEP 440] compliant version numbers,
-which are not compatible with SemVer.
-
-## Installation
-
-Install using [pip], e.g.
-
-```sh
-pip install setuptools_scm_git_semver
-```
+which are not always compatible with SemVer.
 
 ## Usage
 
-To configure `setuptools` to generate SemVer compatible numbers through `setuptools_scm` you could
-use:
+Usage is no different from normal setuptools_scm -- this package just adds 1
+version scheme and local scheme each:
 
-```py
-# setup.py
-from setuptools import setup
-from setuptools_scm_git_semver import parse
+- `plain-version`: Always returns just the (next) version, e.g. "1.2.3", or "0.5.2-rc.2"
 
-setup(
-    ...
-    use_scm_version={
-        'parse': parse,
-        'version_scheme': 'semver',
-        'local_scheme': 'semver'
-    },
-    setup_requires=['setuptools_scm', 'setuptools_scm_git_semver'],
-    ...
-)
+- `time-dot-node`: Returns a local version of +<time>.<node>
 
-# $ python setup.py --version
-# 1.2.4-rc.0.20190927T181700Z+ddb8be7
```

Comment:
Can we retain the `-rc.N` part of the version that is output? This ensures that if we're on a dirty commit after a release, the next version that is output isn't a dirty _release_ but a dirty _release candidate_ which means we're publishing our versions in the right oder (i.e. we shouldn't be publishing release candidates _after_ a release).

In particular:

```
> git tag -a 0.1.2
> python3 test.py
0.1.2
> vim blah.foo
> git add -A
> git commit
[master 6a78f9d] ASDASD
 1 file changed, 1 insertion(+), 1 deletion(-)
> python3 test.py
0.1.3+20211202t115332z.g6a78f9d
```

Should instead output:

```
0.1.3-rc.0+20211202t115332z.g6a78f9d
```

And if we were already at `0.1.3-rc.0` the next one should be `-rc.1`, and so on.